mm2src/coins/nft.rs

```diff
@@ -151,15 +141,20 @@ pub async fn get_nft_transfers(ctx: MmArc, req: NftTransfersReq) -> MmResult<Nft
         };
         let my_address = get_eth_address(&ctx, coin_str).await?;
         let uri_without_cursor = format!(
-            "{}{}/nft/transfers?chain={}&{}&{}",
-            URL_MORALIS, my_address.wallet_address, chain_str, FORMAT_DECIMAL_MORALIS, DIRECTION_BOTH_MORALIS
+            "{}{}{}/nft/transfers?chain={}&{}&{}",
+            req.url,
+            MORALIS_API_ENDPOINT,
+            my_address.wallet_address,
+            chain_str,
+            FORMAT_DECIMAL_MORALIS,
+            DIRECTION_BOTH_MORALIS
         );
```

Comment:
I would prefer doing:

```rs
        let req_url = &req.url;
        let wallet_address = &my_address.wallet_address;
        let uri_without_cursor =
            format!("{req_url}{MORALIS_API_ENDPOINT}{wallet_address}/nft?chain={chain_str}&{FORMAT_DECIMAL_MORALIS}");
```
for readibility on the url building

Reply:
Done